Fixed bug #75

kimi1991 opened the pull request:
Fixed bug #75, added timeout to "takeForcedMeasurement" and a return value to check the result of the function

kimi1991 committed:
Fixed bug #75

Fixed bug #75, added timeout to "takeForcedMeasurement" and a return value to check the result of the function

kimi1991 commented:
Does anybody knows the problem?

kimi1991 commented:
@siddacious any idea why the check failed?

siddacious commented:
hi @kimi1991 , the check failed because `clang-format` decided it wasn't fond of the formatting 🤖 🔤 

I've updated the README to include pointers on how to use `clang-format` to adjust the formatting. Take a look and let me know if you have any more questions:

https://github.com/adafruit/Adafruit_BME280_Library/blob/master/README.md#formatting-and-clang-format

kimi1991 committed:
Changed to clang format

Replace tabs with whitespaces. Changed indentation of function "takeForcedMeasurement"

kimi1991 committed:
Changed to clang format

Added whitespace
